package.json

```diff
@@ -83,11 +85,9 @@
     "ember-welcome-page": "1.0.1",
     "eslint": "3.18.0",
     "faker": "^4.1.0",
-    "jscs": "^3.0.7",
-    "liquid-fire": "0.24.1",
+    "liquid-fire": "^0.27.3",
```

Comment:
We can. If not, it will only get updated when someone adds/removes/updates package.json, otherwise, everything is set to use `--pure-lockfile` (travis, docker, docker-compose, etc)